packages/x-license-pro/src/licenseStatus.ts

```diff
@@ -1,6 +1,8 @@
-export enum LicenseStatus {
+enum LicenseStatus {
   NotFound = 'NotFound',
   Invalid = 'Invalid',
   Expired = 'Expired',
   Valid = 'Valid',
 }
+
+export { LicenseStatus };
```

Comment:
I have a weird compilation error on Codesandbox otherwise when using our ESM bundle as the input.
It gets the following output:

```ts
exports.LicenseStatus = void  0;
Object.defineProperty(exports, "__esModule", {
  value: true
});
let LicenseStatus;
exports.LicenseStatus = LicenseStatus;
(function (LicenseStatus) {
  LicenseStatus["NotFound"] = "NotFound";
  LicenseStatus["Invalid"] = "Invalid";
  LicenseStatus["Expired"] = "Expired";
  LicenseStatus["Valid"] = "Valid";
})(LicenseStatus || (LicenseStatus = {}));
```

And `exports.LicenseStatus` remains undefined (you can test in the console).

The changes in the code allow to init the enum before exporting it, which solves the problem.

Reply:
@michaldudak I think you've experienced similar issue with Codesandbox recently. What was your solution to that?

Reply:
Replace

Reply:
It looks like an error in the Codesandbox' bundler. I created an issue in their repo: https://github.com/codesandbox/codesandbox-client/issues/6435

@flaviendelangle nice work finding a workaround!

Reply:
Always a pleasure to dive into compiled code with 5mn between each test due to the build duration :laughing: